ftl_installer/parser/Parser.py

```diff
@@ -2,6 +2,17 @@
 import argparse
 import ftl_installer
 import logging
+import pprint
+import yaml
+
+class ConfigHandler(object):
+    '''
+    Class to handle the configuration given with `--config` option
+    '''
+
+    def __init__(self, config_file):
+        with open(config_file) as config_file:
+            self.stored_config = yaml.safe_load(config_file)
```

Comment:
This class should probably have a few additional methods that do the following things:
- write the config file out to disk (should use the block format instead of the flow format)
- return a string representation of the config file (converting the config data into a "yaml string")